### PR TITLE
Form CheckBox design-system

### PR DESCRIPTION
**Pull request by JadhavJeet**

### CODE REVIEW

File: src/design-system/formcheckbox.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.dcx-checkbox {
+  color: token('color-text-formcontrol_label');
+  font-weight: token('font-weight-formcontrol_label');
+  font-size: token('font-size-formcontrol_label');
+
```
Comment (jgonza16) [inline src/design-system/formcheckbox.css:5]: We need spacing tokens in checkbox. And i suggest use display flex with gap rule for this.

image

File: src/design-system/formcheckbox.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  & .dcx-checkbox-checkbox--error {
+    color: token('color-text-formcontrol_label-error');
+  }
+
+  & .dcx-checkbox-label--error {
+    color: token('color-text-formcontrol_label-error');
+  }
+
+  & .focusCheckBox:focus {
```
Comment (jgonza16) [inline src/design-system/formcheckbox.css:18]: Three things here, the first one css is not case sensistive use `name-clase` instead `nameClase`, the second one the sahdow is not a token this is not setteble with token, the last one in design system the user shouldn't add custom clas sfor this you can use other css selector like this

```css
& input:focus {
    box-shadow: 0 0 5px #007bff;
  }
&:has(input:focus) {
    box-shadow: 0 0 5px #007bff;
  }
```

**Comment by jgonza16** [comment]
the chekbox look like native checkbox maybe we need apply material look and feel https://m2.material.io/components/checkboxes#usage

File: src/design-system/formcheckbox.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  color: token('color-text-formcontrol_label');
+  font-weight: token('font-weight-formcontrol_label');
+  font-size: token('font-size-formcontrol_label');
+
+  & .dcx-checkbox-container--error {
+    color: token('color-text-formcontrol_label-error');
+  }
+
+  & .dcx-checkbox-checkbox--error {
+    color: token('color-text-formcontrol_label-error');
```
Comment (jgonza16) [inline src/design-system/formcheckbox.css:11]: i think that the error styles should be equal in label and input

image

**Review by jgonza16 — DISMISSED** [review]

**Comment by daniele-zurico** [comment]
@jgonza16 there's a minor bug.... when the checkbox is disabled when you over it it changes the color instead I think it should stay the same

**Review by daniele-zurico — APPROVED** [review]

**Pull request merged**